SAP/luigi
# Add ie11 support

**Description**

Changes proposed in this pull request:

- Add legacy bundle for both Luigi Core and Luigi Client
- Functionality in evergreen browsers not affected (current build stays as it was)
- Fix issues on IE11 even when using legacy bundle
- Add _.browserslistrc_ file for consistent browser support
- Update fiori-fundamentals to newest version (1.6.0) which fixes some bugs
- Redefine in App.html the fiori-fundamental variables used in Luigi Core to avoid issues on css transformation for legacy bundle.
- Add docu for legacy bundles
- Update Webpack loaders for Luigi Core

**How to test it**
- If you don't have IE11 installed in your machine, download a virtual machine with IE11 from here for the platform of your choice: https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/
- You can use this example app by placing this folder into the Luigi Core folder: 
[test-ie11.zip](https://github.com/SAP/luigi/files/3325178/test-ie11.zip)
- From your terminal, go to the Luigi Core folder and start a web server by running ```npx ws```. You will get an IP address on your terminal that you can use to run the app in IE11 in the VM.
- In another tab in your terminal, open Luigi root folder and run ```lerna run bundle```. This will create both regular and legacy bundles for both Core and Client"
- Start the virtual machine, open IE11 and go to _YOUR-IP:YOUR-PORT/test-ie11_
- You can play around with the imports on _test-ie11/index.html_ to see the styling differences among legacy and not legacy bundle.

**Related issue(s)**
